main.py

```diff
@@ -11,33 +12,41 @@ class User(BaseModel):
     email: str
 
 
-@app.post("/users/")
-async def create_user(request: Request):
+@asynccontextmanager
+async def get_db():
+    conn = await asyncpg.connect(settings.DATABASE_URL)
     try:
-        data = await request.json()
-        user = User(**data)
-        user_data = user.dict()
-        result = users_collection.insert_one(user_data)
+        yield conn
+    finally:
+        await conn.close()
 
-        return {"id": str(result.inserted_id)}
-    except Exception as e:
 
+async def create_customer(conn, name: str, email: str):
+    query = """
+    INSERT INTO customers (name, email)
+    VALUES ($1, $2)
+    RETURNING id
+    """
+    try:
+        async with conn.transaction():
+            user_id = await conn.fetchval(query, name, email)
+        return user_id
+    except Exception as e:
         raise HTTPException(status_code=400, detail=str(e))
 
 
-class UpdateUser(BaseModel):
-    name: str = None
-    email: str = None
-    age: int = None
-
+@app.post("/customers/")
+async def create_user_endpoint(user: User):
+    try:
+        async with get_db() as db:
+            user_id = await create_customer(db, user.name, user.email)
+        return {"id": user_id, "name": user.name, "email": user.email}
+    except HTTPException as e:
+        raise e
+    except Exception as e:
+        raise HTTPException(status_code=400, detail=str(e))
 
-@app.get("/users/{user_id}")
-async def get_user(user_id: str):
-    user = users_collection.find_one({"_id": ObjectId(user_id)})
-    if user:
-        return {
-            "name": user["name"],
-            "email": user["email"],
-        }
 
-    raise HTTPException(status_code=404, detail="User not found")
+@app.get("/users")
+async def get_users():
+    return "hello world"
```

Comment:
Just putting it on record here, this was done for testing I suppose, then we'll change the arc of the project.

Reply:
yeah, that's right!!!